app/controlplane/pkg/data/group.go

```diff
@@ -573,6 +577,122 @@ func (g GroupRepo) UpdateMemberMaintainerStatus(ctx context.Context, orgID uuid.
 	})
 }
 
+// ListProjectsByGroup retrieves a list of projects that a group is a member of with pagination.
+func (g GroupRepo) ListProjectsByGroup(ctx context.Context, orgID uuid.UUID, groupID uuid.UUID, visibleProjectIDs []uuid.UUID, paginationOpts *pagination.OffsetPaginationOpts) ([]*biz.GroupProjectInfo, int, error) {
+	if paginationOpts == nil {
+		paginationOpts = pagination.NewDefaultOffsetPaginationOpts()
+	}
+
+	// Check the group exists in the organization
+	existingGroup, err := g.data.DB.Group.Query().
```

Comment:
This query was already done by the caller, right?

Reply:
I'm curious why it needs the group since this is just listing projects. The group is already known, because it's what originated the call, right?

Reply:
We might not need it for now